test/tests/brakeman.rb

```diff
@@ -278,6 +278,49 @@ def test_output_format
       Reports models which do not use attr_restricted and warns on models that use attr_protected\nCheckModelSerialize           Report uses of serialize in versions vulnerable to CVE-2013-0277\nCheckNestedAttributes         Checks for nested attributes vulnerability in Rails 2.3.9 and 3.0.0\nCheckNumberToCurrency         Checks for number helpers XSS vulnerabilities in certain versions\nCheckQuoteTableName           Checks for quote_table_name vulnerability in versions before 2.3.14 and 3.0.10\nCheckRedirect                 Looks for calls to redirect_to with user input as arguments\nCheckRegexDoS                 Searches regexes including user input\nCheckRender                   Finds calls to render that might allow file access\nCheckRenderDoS                Warn about denial of service with render :text (CVE-2014-0082)\nCheckRenderInline             Checks for cross site scripting in render calls\nCheckResponseSplitting        Report response splitting in Rails 2.3.0 - 2.3.13\nCheckSafeBufferManipulation   Check for Rails versions with SafeBuffer bug\nCheckSanitizeMethods          Checks for versions with vulnerable sanitize and sanitize_css\nCheckSelectTag                Looks for unsafe uses of select_tag() in some versions of Rails 3.x\nCheckSelectVulnerability      Looks for unsafe uses of select() helper\nCheckSend                     Check for unsafe use of Object#send\nCheckSendFile                 Check for user input in uses of send_file\nCheckSessionManipulation      Check for user input in session keys\nCheckSessionSettings          Checks for session key length and http_only settings\nCheckSimpleFormat             Checks for simple_format XSS vulnerability (CVE-2013-6416) in certain versions\nCheckSingleQuotes             Check for versions which do not escape single quotes (CVE-2012-3464)\nCheckSkipBeforeFilter         Warn when skipping CSRF or authentication checks by default\nCheckSQL                      Check for SQL injection\nCheckSQLCVEs                  Checks for several SQL CVEs\nCheckSSLVerify                Checks for OpenSSL::SSL::VERIFY_NONE\nCheckStripTags                Report strip_tags vulnerabilities CVE-2011-2931 and CVE-2012-3465\nCheckSymbolDoSCVE             Checks for versions with ActiveRecord symbol denial of service vulnerability\nCheckTranslateBug             Report XSS vulnerability in translate helper\nCheckUnsafeReflection         Checks for unsafe reflection\nCheckValidationRegex          Report uses of validates_format_of with improper anchors\nCheckWithoutProtection        Check for mass assignment using without_protection\nCheckXMLDoS                   Checks for XML denial of service (CVE-2015-3227)\nCheckYAMLParsing              Checks for YAML parsing vulnerabilities (CVE-2013-0156)\nCheckSymbolDoS                Checks for symbol denial of service\nCheckUnscopedFind             Check for unscoped ActiveRecord queries\nCheckWeakHash                 Checks for use of weak hashes like MD5\n"
```

Comment:
good suggestion. I have updated the 2 tests to get the list of checks from Brakeman::Checks. There is currently a private method: get_check_name that is in Brakeman::Checks - which has the same code: to_s.split("::").last that I am using in the test. Do you have any objections if I make the method public to call it from the test? Or is the test code OK?